transformation/voltage-normalization/README.md

```diff
@@ -0,0 +1,224 @@
+# Normalize Voltage Levels
+This cookbook is about normalizing voltage levels for energy monitoring.
+
+### Use-case
+> In an energy monitoring system, voltage readings are collected from various sensors and devices. These sensors may use different voltage scales. The analytics services require voltage data normalized to a standardized scale for consistent analysis.
+
+For this use-case, a script, a policy, and a schema are required.
+
+### Schema
+
+To begin with, consider the following plain JSON schema (according to [JSON Schema](https://json-schema.org/)):
+
+`schema-from-sensor.json`:
+```json
+{
+  "type": "object",
+  "properties": {
+    "voltage": {
+      "type": "number"
+    },
+    "sensor_type": {
+      "type": "integer",
+      "enum": [1, 2, 3]
```

Comment:
Question since I'm not very familiar with JSONSchema: is `"type": "integer"` required here? Without it, would `1.0` for example be allowed?

Reply:
Just tried it. Yes, without `"type": "integer"` `1.0` is allowed and it is converted to `1` - so the `enum` is met. However we probably do not want `1.0` to be allowed this way - so I will leave `"type": "integer"` how it is now.